doc/whats_new/v1.0.rst

```diff
@@ -146,6 +146,10 @@ Changelog
   as `base_estimator` in ::class:`calibration.CalibratedClassifierCV`.
   :pr:`20087` by :user:`Clément Fauchereau <clement-f>`.
 
+- |Fix| Fixed ::class:`calibration.CalibratedClassifierCV` to handle correctly
```

Comment:
We would need to move this fix under a section 1.0.1 because it will be a bug fix that will be integrated in the next bug fix release

Reply:
v1.0.1.rst is not created yet.

I should wait for its creation to add:

- |Fix| Fixed :class:`calibration.CalibratedClassifierCV` to handle correctly
  `sample_weight` when `ensemble=False`.
  :pr:`20638` by :user:`Julien Bohné <JulienB-78>`.

or there is another way?

Reply:
> v1.0.1.rst is not created yet.

It should be in `v1.0.rst` file. It will only be a new section. You can have a look at the `0.24.rst` file to see the `0.24.2` section.

Reply:
I have created the 1.0.1 section.

Some builds fail but I guess it is unrelated. Should I just wait?

Reply:
Indeed, I relaunch the build. It was due to an error at the install of the docker file. This is a random failure on Azure side I assume.